lgn-messages/src/types/v1/query/mod.rs

```diff
@@ -14,6 +14,9 @@ pub mod tasks;
 
 pub const ROUTING_DOMAIN: &str = "sc";
 
+pub const NUM_CHUNKS: usize = 5;
+pub const NUM_ROWS: usize = 5;
```

Comment:
Could you please add a documentation string describing what these constants mean? Should they be kept in sync with constants in MP2, or are DQ/Worker specific?

Reply:
Those constants should be only const generic in MP2, so they can be freely set on DQ. 
The proper values should be tuned with proper benchmarks on actual workers: higher constant values means less proofs to be generated to prove the same number of rows, but on the other hand the circuits get bigger. So we need to find the best tradeoff between time/resources to compute a single proof (which depends on circuit size) and the overall number of proofs to be done in order to prove a query. My gut feeling is that we should increase those constants until the proving times/memory consumption for a single proof on a reference worker machine becomes too high